### ISSUE TITLE
Admin panel revision/question

### ISSUE THREAD

**Issue by catyneis** [YoodleLLC/FDI#13]
In the Admin Panel, delete the field “last sent.”

Will you please confirm “first access” and “last access” will reflect dates/times of when someone has used a username and password to access a packet (not when an internal person accesses the packet from the Admin Panel).

**Comment by catyneis** [comment]
Also: 
Currently in the admin panel we have the option to email the link. There should be a control where the user is notified that the packet has been set up and they are able to access it. Can you explain how that works please?
Should we remove this from the Admin Panel once the new Client Connect login goes live?

**Comment by timmccoy2yoodle** [comment]
This is correct -  "Will you please confirm “first access” and “last access” will reflect dates/times of when someone has used a username and password to access a packet"

Looking into the other questions.

**Comment by catyneis** [comment]
Please provide direction on the notification setup. Let me know if we need a call to walk through it.
